openstack/02b-MAChigh/neighbors.c

```diff
@@ -681,6 +682,14 @@ void registerNewNeighbor(open_addr_t *address,
                 if (rssi < GOODNEIGHBORMINRSSI) {
                     break;
                 }
+                open_addr_t addr_16;

```

Comment:
the convention we use is to declare variables at the beginning of the function, one var per line. applies throughout.

Reply:
So you want me to move it up to line 669? AFAIU putting it inside the iff reduces its scope to where it's needed, but I can move it up if you prefer.

Reply:
Moved up as suggested